components/remote-environment-broker/README.md

```diff
@@ -39,6 +43,19 @@ After normalization, the **displayName** contains only lowercase characters, num
 
 Before each commit, use the `before-commit.sh` script, which tests your changes.
 
+### Use environment variables
+
+| Name | Required | Default | Description |
+|-----|---------|--------|------------|
+|**APP_PORT** | false | 8080 | The port on which the HTTP server listen. | 
+|**APP_BROKER_RELIST_DURATION_WINDOW** | true | - | Synchronize REB in Service Catalog (if needed) at most once per this period |
```

Comment:
But it not reflect what I meant.
If Remote Environment is added, then we synchronize REB in the Service Catalog. But we do it not often than this parameter, so if 2 new Remote Environments will be added, one immediatelly another, only one SYNC will be sent. 
In case no changes in Remote Environment, we don't SYNC at all.

Reply:
Time period after which the REB synchronizes with the Service Catalog if a new Remote Environment is added. In case more than one Remote Environment is added, synchronization is performed only once.